pkgs/tools/inputmethods/netevent/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, docutils }:
+
+stdenv.mkDerivation rec {
+  pname = "netevent";
+  version = "20201018";
+
+  src = fetchFromGitHub {
+    owner = "Blub";
+    repo = "netevent";
+    rev = "ddd330f0dc956a95a111c58ad10546071058e4c1";
+    sha256 = "0myk91pmim0m51h4b8hplkbxvns0icvfmv0401r0hw8md828nh5c";
+  };
+
+  buildInputs = [ docutils ];
+
+  outputs = [ "out" "doc" "man" ];
+
+  RST2MAN = "rst2man";
+
+  configurePhase = ''
+    ./configure
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv netevent $out/bin/
+
+    mkdir -p $man/share/man/man1
+    mv doc/netevent.1 $man/share/man/man1/netevent.1
+
+    mkdir -p $doc/share/doc/netevent
+    cp doc/netevent.rst $doc/share/doc/netevent/netevent.rst
+  '';
```

Comment:
This all seems unnecessary, seems to be done by the package itself if you set --prefix appropriately in configure.

Reply:
When setting the --prefix in the configure (which is not a classical configure), the data does not end up in the good directory, so I choose to do it manually.

Reply:
From looking at the configure script, the directories seem fine if you just set prefix=$out. What's wrong concretely?

Reply:
Here what I get:

```
 nix-build -A netevent
these derivations will be built:
  /nix/store/bxh7y6n33n4n46cyjn8fldbvl2idif1v-netevent-20201018.drv
building '/nix/store/bxh7y6n33n4n46cyjn8fldbvl2idif1v-netevent-20201018.drv'...
unpacking sources
unpacking source archive /nix/store/1k2ming61x2ga1kz43x39c5lg4q0ph1a-source
source root is source
patching sources
configuring
Checking for rst2man...rst2man
Checking for UI_DEV_SETUP (kernel >= 4.4)...ok
building
build flags: SHELL=/nix/store/k8p54jg8ipvnfz435mayf5bnqhw4qqap-bash-4.4-p23/bin/bash
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/main.o src/main.cpp -MMD -MT src/main.o -MF src/main.d
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/daemon.o src/daemon.cpp -MMD -MT src/daemon.o -MF src/daemon.d
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/writer.o src/writer.cpp -MMD -MT src/writer.o -MF src/writer.d
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/reader.o src/reader.cpp -MMD -MT src/reader.o -MF src/reader.d
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/socket.o src/socket.cpp -MMD -MT src/socket.o -MF src/socket.d
g++ -g -Wall -Werror -Wno-unknown-pragmas  -std=c++14 -c -o src/bitfield.o src/bitfield.cpp -MMD -MT src/bitfield.o -MF src/bitfield.d
g++ -g  -o netevent src/main.o src/daemon.o src/writer.o src/reader.o src/socket.o src/bitfield.o
rst2man doc/netevent.rst doc/netevent.1
installing
install flags: SHELL=/nix/store/k8p54jg8ipvnfz435mayf5bnqhw4qqap-bash-4.4-p23/bin/bash install
make install-do
make[1]: Entering directory '/build/source'
install -dm755
install: missing file operand
Try 'install --help' for more information.
make[1]: *** [Makefile:67: install-do] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:64: install] Error 2
builder for '/nix/store/bxh7y6n33n4n46cyjn8fldbvl2idif1v-netevent-20201018.drv' failed with exit code 2
error: build of '/nix/store/bxh7y6n33n4n46cyjn8fldbvl2idif1v-netevent-20201018.drv' failed
```

Given the state of the configure file, I prefer to move things manually around.

Reply:
Any comment @catern ? I'm not really sure how I should proceed now, as it's my first package request i there, and because the points you made are good, but at the same time I'm not sure they can be addressed by the package management. Do you think I should resolve the conversation? Do I have to wait for more reviews?

Reply:
Since you're working off the master branch of the package anyway, maybe you can try and fix the configure script with a patch upstream.